pkg/endpoint/endpoint.go

```diff
@@ -67,102 +67,11 @@ import (
 )
 
-var (
-	//IPv4Enabled can be set to false to indicate IPv6 only operation
-	IPv4Enabled = true
-)
-
 const (
-	OptionConntrackAccounting = "ConntrackAccounting"
-	OptionConntrackLocal      = "ConntrackLocal"
-	OptionConntrack           = "Conntrack"
-	OptionDebug               = "Debug"
-	OptionDebugLB             = "DebugLB"
-	OptionDropNotify          = "DropNotification"
-	OptionTraceNotify         = "TraceNotification"
-	OptionNAT46               = "NAT46"
-	OptionIngressPolicy       = "IngressPolicy"
-	OptionEgressPolicy        = "EgressPolicy"
-	AlwaysEnforce             = "always"
-	NeverEnforce              = "never"
-	DefaultEnforcement        = "default"
-
 	maxLogs = 256
 )
 
 var (
-	OptionSpecConntrackAccounting = option.Option{
-		Define:      "CONNTRACK_ACCOUNTING",
-		Description: "Enable per flow (conntrack) statistics",
-		Requires:    []string{OptionConntrack},
-	}
-
-	OptionSpecConntrackLocal = option.Option{
-		Define:      "CONNTRACK_LOCAL",
-		Description: "Use endpoint dedicated tracking table instead of global one",
-		Requires:    []string{OptionConntrack},
-	}
-
-	OptionSpecConntrack = option.Option{
-		Define:      "CONNTRACK",
-		Description: "Enable stateful connection tracking",
-	}
-
-	OptionSpecDebug = option.Option{
-		Define:      "DEBUG",
-		Description: "Enable debugging trace statements",
-	}
-
-	OptionSpecDebugLB = option.Option{
-		Define:      "LB_DEBUG",
-		Description: "Enable debugging trace statements for load balancer",
-	}
-
-	OptionSpecDropNotify = option.Option{
-		Define:      "DROP_NOTIFY",
-		Description: "Enable drop notifications",
-	}
-
-	OptionSpecTraceNotify = option.Option{
-		Define:      "TRACE_NOTIFY",
-		Description: "Enable trace notifications",
-	}
-
-	OptionSpecNAT46 = option.Option{
-		Define:      "ENABLE_NAT46",
-		Description: "Enable automatic NAT46 translation",
-		Requires:    []string{OptionConntrack},
-		Verify: func(key string, val bool) error {
-			if !IPv4Enabled {
-				return fmt.Errorf("NAT46 requires IPv4 to be enabled")
-			}
-			return nil
-		},
-	}
-
-	OptionIngressSpecPolicy = option.Option{
-		Define:      "POLICY_INGRESS",
-		Description: "Enable ingress policy enforcement",
-	}
-
-	OptionEgressSpecPolicy = option.Option{
-		Define:      "POLICY_EGRESS",
-		Description: "Enable egress policy enforcement",
-	}
-
-	EndpointMutableOptionLibrary = option.OptionLibrary{
-		OptionConntrackAccounting: &OptionSpecConntrackAccounting,
-		OptionConntrackLocal:      &OptionSpecConntrackLocal,
-		OptionConntrack:           &OptionSpecConntrack,
-		OptionDebug:               &OptionSpecDebug,
-		OptionDebugLB:             &OptionSpecDebugLB,
-		OptionDropNotify:          &OptionSpecDropNotify,
-		OptionTraceNotify:         &OptionSpecTraceNotify,
-		OptionNAT46:               &OptionSpecNAT46,
-		OptionIngressPolicy:       &OptionIngressSpecPolicy,
-		OptionEgressPolicy:        &OptionEgressSpecPolicy,
-	}
-
 	EndpointOptionLibrary = option.OptionLibrary{}
```

Comment:
This can be an uninitialised var now, which would make it clearer that the empty one isn't intended to be used (init will always run below...)